src/sif/pedestriancost.cc

```diff
@@ -469,6 +474,26 @@ class PedestrianCost : public DynamicCost {
     return c;
   }
 
+public:
+  /**
+   * Override the base transition cost to not add maneuver penalties onto transit edges.
+   * Base transition cost that all costing methods use. Includes costs for
+   * country crossing, boarding a ferry, toll booth, gates, entering destination
+   * only, alleys, and maneuver penalties. Each costing method can provide different
+   * costs for these transitions (via costing options).
+   * @param node Node at the intersection where the edge transition occurs.
+   * @param edge Directed edge entering.
+   * @param pred Predecessor edge information.
+   * @param idx  Index used for name consistency.
+   * @return Returns the transition cost (cost, elapsed time).
+   */
+  sif::Cost base_transition_cost(const baldr::NodeInfo* node,
```

Comment:
It took me 2 days to understand that there were two implementations totally identical, I should've added the BSS cost in both methods... So I did a tiny refactor...